internal/util/bundleutil/bundleutil.go

```diff
@@ -0,0 +1,260 @@
+// Copyright 2021 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bundleutil
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"sort"
+	"text/template"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+
+	"github.com/operator-framework/operator-sdk/internal/flags"
+	"github.com/operator-framework/operator-sdk/internal/util/projutil"
+)
+
+var (
+	defaultMetadataDir          = "metadata"
+	defaultManifestDir          = "manifests"
+	defaultBundleDockerfilePath = "bundle.Dockerfile"
+)
+
+// BundleMetaData contains the required metadata to build bundles and images.
+type BundleMetaData struct {
+	// BundleDir refers to the directory where generated bundles are to be written.
+	BundleDir string
+
+	// The PackageName of the operator bundle.
+	PackageName string
+
+	// Channels and DefaultChannel the operator shoud be subscribed to.
+	Channels       string
+	DefaultChannel string
+
+	// BaseImage name to build bundle image.
+	BaseImage string
+
+	// BuildCommand to run while building image.
+	BuildCommand string
+
+	// PackageManifestPath where the input manifests are present.
+	PkgmanifestPath string
+
+	// IsScoreconfigPresent when set to true includes scorecard config annotations
+	// in bundle metadata.
+	IsScoreconfigPresent bool
+
+	// Other labels to be added in CSV.
+	OtherLabels map[string]string
+}
+
+// values to populate bundle metadata/Dockerfile.
+type annotationsValues struct {
+	BundleDir           string
+	PackageName         string
+	Channels            string
+	DefaultChannel      string
+	OtherLabels         []string
+	ScorecardConfigPath bool
```

Comment:
Nit: make this field sound like a bool
```suggestion
	IsScorecardConfigPresent bool
```